libraries/AP_OSD/AP_OSD.h

```diff
@@ -215,9 +219,7 @@ class AP_OSD_Screen : public AP_OSD_AbstractScreen
     void draw_aspd1(uint8_t x, uint8_t y);
     void draw_aspd2(uint8_t x, uint8_t y);
     void draw_vspeed(uint8_t x, uint8_t y);
-#if HAL_PLUSCODE_ENABLE
```

Comment:
why is this changing things related to pluscode?

Reply:
historical....havent the foggiest....do I have to figure this one out?

Reply:
will fix

Reply:
This still seems to be playing around with PLUSCODE stuff?